modular_skyrat/modules/colony_fabricator/sound/attributions.txt

```diff
@@ -12,3 +12,7 @@ modular_skyrat/modules/colony_fabricator/sound/conditioner/conditioner_(1 - 4).w
 The following sounds are from https://pixabay.com/sound-effects/welder-3-54547/
 
 modular_skyrat/modules/colony_fabricator/sound/arc_welder/arc_welder.ogg
+
+The following sounds are from https://pixabay.com/sound-effects/sos-signal-137144/
+
+modular_skyrat/modules/colony_fabricator/sound/morse_signal.wav
```

Comment:
I haven't see it, where does this sound actually come into play in all of this?

Reply:
This is the headset's radio sound effect, the one it makes when you speak over comms